Oneirocom/Magick
# Node ux improvements

## What Changed:

- restyled nodes
- Improved node selection highlight
- improved connection lines
  - unselected - medium thickness
  - select node - node connections appear bolder and glow
- added connection glow
- sockets get larger on hover
- connections connected to hovered socket highlight
- you can deselect nodes now by clicking on the editor
- modified styling of comments
- added our brand font Berkely mono as a remote font resource (can't check into repo due to licensing)

## How to test:

- Ensure nodes still turn green/red on run
- select a node and follow its connections
- hover over sockets to highlight a connection
- select a node and click on the editor area of deselect again

## Additional information:

- There are a few known bugs
  - when the connection is horizontal it disappears
  - the glow appears to be cut off at a bounding box around the connections
  - if you select a node and hover over its socket, the connection stops being high-lighted
- I suspect the first two bugs are related, and the SVG view box needs to be modified somehow.  I am not great with svg so figured I would submit this now and if someone else gets to it before me, awesome.
-
